examples/ui_actions_explorer/public/hello_world_example.tsx

```diff
@@ -69,25 +69,33 @@ export const HelloWorldExample = ({
           onChange={(e) => {
             setIsChecked(e.target.checked);
             if (e.target.checked) {
-              const dynamicAction = createAction({
-                id: DYNAMIC_ACTION_ID,
-                type: ACTION_HELLO_WORLD,
-                getDisplayName: () => 'Say hello to Waldo',
-                execute: async () => {
-                  const overlay = overlays.openModal(
-                    toMountPoint(
-                      <EuiModalBody>
-                        <EuiText data-test-subj="dynamicHelloWorldActionText">Hello Waldo</EuiText>{' '}
-                        <EuiButton data-test-subj="closeModal" onClick={() => overlay.close()}>
-                          Close
-                        </EuiButton>
-                      </EuiModalBody>,
-                      rendering
-                    )
-                  );
-                },
-              });
-              uiActionsStartService.addTriggerAction(HELLO_WORLD_TRIGGER_ID, dynamicAction);
+              uiActionsStartService.addTriggerActionAsync(
+                HELLO_WORLD_TRIGGER_ID,
+                DYNAMIC_ACTION_ID,
+                async () => {
+                  const { createAction } = await import('@kbn/ui-actions-plugin/public');
```

Comment:
I don't think this async import makes much sense. 🤔 This bundle is anyway should be included in the main bundle